assets/js/base/context/hooks/cart/test/use-store-cart.js

```diff
@@ -17,6 +17,7 @@ jest.mock( '../../../providers/editor-context', () => ( {
 } ) );
 
 jest.mock( '@woocommerce/block-data', () => ( {
+	...jest.requireActual( '@woocommerce/block-data' ),
```

Comment:
What is the purpose of this? Is this something we should do for other mocks in our tests in another PR, I see not all of them use them.

Reply:
If this line wasn't there, any exported constant that wasn't mocked, would be undefined. In this case, all new `EMPTY_XYZ` arrays would be undefined because they are not explicitly added to the mock.

I'm not sure if there is any drawback of doing this change to all other mocks, but it makes tests to be closer to the way production code runs, so in general I think it's better to have it, even if not needed.